src/Symfony/Component/DependencyInjection/Container.php

```diff
@@ -63,7 +63,11 @@ class Container implements ResettableContainerInterface
      */
     protected $parameterBag;
 
+    /**
+     * @var object[]
+     */
```

Comment:
To be reverted IMO. (It's indexed by a string anyway, and IIRC `object[]` is only valid for sequential arrays)